test/testcafe/spec/AuthenticatorIdPView_spec.js

```diff
@@ -0,0 +1,74 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import IdPAuthenticatorPageObject from '../framework/page-objects/IdPAuthenticatorPageObject';
+import xhrEnrollIdPAuthenticator from '../../../playground/mocks/data/idp/idx/authenticator-enroll-idp.json';
+import xhrVerifyIdPAuthenticator from '../../../playground/mocks/data/idp/idx/authenticator-verification-idp.json';
+
+
+const logger = RequestLogger(/introspect/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+const enrollMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrEnrollIdPAuthenticator)
+  .onRequestTo('http://localhost:3000/idp/idx/credential/enroll')
+  .respond(xhrEnrollIdPAuthenticator)
+  .onRequestTo('http://localhost:3000/sso/idps/0oa69chx4bZyx8O7l0g4?stateToken=02TptqPN4BOLIwMAGUVLPlZVJEnONAq7xkg19dy6Gk')
+  .respond('<html><h1>An external IdP login page for testcafe testing</h1></html>');
+
+const verifyMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrVerifyIdPAuthenticator)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge')
+  .respond(xhrVerifyIdPAuthenticator)
+  .onRequestTo('http://localhost:3000/sso/idps/0oa69chx4bZyx8O7l0g4?stateToken=02TptqPN4BOLIwMAGUVLPlZVJEnONAq7xkg19dy6Gk')
+  .respond('<html><h1>An external IdP login page for testcafe testing</h1></html>');
+
+async function setup(t) {
+  const pageObject = new IdPAuthenticatorPageObject(t);
+  await pageObject.navigateToPage();
+
+  const { log } = await t.getBrowserConsoleMessages();
+
+  await t.expect(log.length).eql(3);
```

Comment:
heard someone is improving how we assert all these log messages... you should take a look at that person's PR, it will greatly improve how we do this! 😉